flow/scripts/floorplan.tcl

```diff
@@ -87,6 +87,38 @@ remove_buffers
 
 source $::env(SCRIPTS_DIR)/report_metrics.tcl
 report_metrics "floorplan final"
+set num_instances [llength [get_cells -hier *]]
+puts "number instances before restructure is $num_instances"
+puts "Design Area before restructure"
+report_design_area
```

Comment:
Have you considered just including the number of instances in the report_metics.tcl file? Then you would only have to call `report_metics "floorplan post-restructure"` to get the stats you are looking for, `report_design_area` is already collected as part of that call?

Reply:
It looks like quite a baggage to include. Since this is optional feature, don't want to bombard with too many metrics. As feature matures and becomes default, we can include full metrics.

Reply:
There is no point to print this if you aren't going to restructure.

Reply:
Ok, I can put it inside if statement. Then when we generate variant comparison, we'll get zero for non-restructure variant.